### ISSUE TITLE
Selenium-side-runner | additional parameters Validation Warning

### ISSUE THREAD

**Issue by sibilkov** [SeleniumHQ/selenium-ide#367]
Hi, I'm trying to run tests through the SSR and get this-like message for every additional parameter:

```
● Validation Warning:

  Unknown option "testEnvironmentOptions.server" with value "http://localhost:4444/wd/hub" was found.
  This is probably a typing mistake. Fixing it will remove this message.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

But what is interesting, apparently, all the parameters are applied. Why i get this message?

And olso, all tests (even most easyest) takes too many time for executing. Test which takes 5s in Selenium IDE takes sometimes 45s in SSR. (Often i get WebDriverError: New session attempts retry count exceeded !)
SSR v. 3.2.4
Im using latest Selenoid if its important.

**Comment by M3ssman** [comment]
Yep, there's an closed Issue #354 on that topic. The warnings result from an outdated underlying Library and are not business of sider-runner.

@corevo 
Better re-open #354 and label it with state waiting for response or so, before the next Issue arrives with this?

**Comment by corevo** [comment]
Duplicate of #354 

I downgraded `jest` on the next release so this warning will not repeat itself.  
Once `jest` will merge the pr about it, I will upgrade it again.
